pep-0639.rst

```diff
@@ -300,13 +267,6 @@ for the purposes of this PEP (``Syn:``).
   and ``LicenseRef-Proprietary`` strings. Examples: ``MIT``, ``GPL-3.0-only``
 
 **Project** *(Sub: Project Source Tree, Installed Project)*
```

Comment:
Minor -- Would it make sense to link the user guide definition here? You expand the definition with source tree vs installed below, which assumes implicit knowledge of the "project" definition. You do explicitly mention you're using project by reference above though, so this is a ease of reading suggestion rather than more critical.

Reply:
Hmm, I thought about this as I do this here and in Distribution Package as well, but I didn't end up doing it initially. I'll do that now.

Reply:
BTW, we really need to fix the margins of the definition lists in the Sphinx theme. There is almost no margin/padding between definitions, much less than even normal `<p>`s, making it very hard to read, while there is at least the normal (adequate) spacing if the definition has multiple paragraphs. For now I could hack it with `|`, but I really hate resorting to embedding such style hints in the source markup.

Reply:
#2300 so this doesn't get lost